agent/consul/leader_connect_test.go

```diff
@@ -512,8 +512,6 @@ func TestLeader_SecondaryCA_TransitionFromPrimary(t *testing.T) {
 func TestLeader_SecondaryCA_UpgradeBeforePrimary(t *testing.T) {
 	t.Parallel()
 
-	maxRootsQueryTime = 500 * time.Millisecond
```

Comment:
I think this is still needed, because it changes how long the roots endpoint is blocking before it returns in the test.

Reply:
I added it to the server configurations. Hopefully that does the trick.